src/at_tools/att_patch/unified_diff_format.rs

```diff
@@ -841,7 +845,8 @@ mod tests {
 @@ ... @@
 ```
 Another text"#;
-        let result = UnifiedDiffFormat::parse_message(input).await.expect(
+        let (gcx, _, _, _) = global_context::tests::create_mock_global_context().await; 
```

Comment:
I don't have to create mock gcx, you can just create default privacy_settings

Reply:
I can not come up with a simpler way than mocking global context. 

I think it is better that the function receives the global context and handles the part of awaiting to get the config it needs, therefore the param should be the global context.

I tried to create a simpler version of global context for the tests, but rust is smart enough to detect is not the same type, and I don't have any more ideas, chat did not provided a better idea either.